validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorConfig.java

```diff
@@ -243,5 +300,38 @@ private void validateKeyStoreFilesAndPasswordFilesConfig() {
         throw new InvalidConfigurationException(errorMessage);
       }
     }
+
+    private void validateExternalSignerUrlAndPublicKeys() {
+      if (validatorExternalSignerPublicKeys == null
+          || validatorExternalSignerPublicKeys.isEmpty()) {
+        return;
+      }
+
+      if (validatorExternalSignerUrl == null) {
+        final String errorMessage =
+            "Invalid configuration. '--validators-external-signer-url' and '--validators-external-signer-public-keys' must be specified together";
+        throw new InvalidConfigurationException(errorMessage);
+      }
+    }
+
+    private void validateExternalSignerKeystoreAndPasswordFileConfig() {
+      if (validatorExternalSignerKeystore == null
+          && validatorExternalSignerKeystorePasswordFile == null) {
+        return;
+      }
+      final String errorMessage =
+          "Invalid configuration. '--validators-external-signer-keystore' and '--validators-external-signer-keystore-password-file' must be specified together";
+      throw new InvalidConfigurationException(errorMessage);
+    }
+
+    private void validateExternalSignerTruststoreAndPasswordFileConfig() {
+      if (validatorExternalSignerTruststore == null
+          && validatorExternalSignerTruststorePasswordFile == null) {
+        return;
+      }
+      final String errorMessage =
+          "Invalid configuration. '--validators-external-signer-truststore' and '--validators-external-signer-truststore-password-file' must be specified together";
+      throw new InvalidConfigurationException(errorMessage);
```

Comment:
to me, this looks to say that they must both be null, or it's invalid configuration?